gapic-generator-java/src/main/java/com/google/api/generator/gapic/composer/samplecode/ServiceClientHeaderSampleComposer.java

```diff
@@ -342,7 +342,7 @@ public static Sample composeTransportSample(
         RegionTag.builder()
             .setServiceName(service.name())
             .setRpcName(rpcName)
-            .setOverloadDisambiguation("setCredentialsProvider")
+            .setOverloadDisambiguation("composeTransport")
```

Comment:
@blakeli0 I believe the name for this sample was wrong. It should be an example showcasing how to switch to the non-default transport (HttpJson).